lib/rucio/transfertool/globusLibrary.py

```diff
@@ -21,7 +21,7 @@
 import os
```

Comment:
nitpick comment:

we generally don't use camelCase. any strong opinion to keep it like that?

Reply:
Yes let's change that.  In fact, this separate globusLibrary.py is really a relic from when I was trying to do this outside Rucio.  These methods could probably be rolled into ./lib/rucio/transfertool/globus.py  What do you think?  (And it's rarely nitpicky when it's correct.)

Reply:
The globus library is a bit special, since you are also using it from the globus protocol. Since the client distribution does include protocols, but not the `transfertool/` tree, the library should probably be put to a common place. E.g. `lib/rucio/common/`